examples/sound/portaudio-sine.bas

```diff
@@ -0,0 +1,82 @@
+''
+'' PortAudio Sine wave example
+''
+'' Requires libportaudio2
+
+#include "portaudio.bi"
+
+Const Pi as Double = 3.14159265358979323846
+Const SampleRate as Integer = 44100
+Const SineSamples as Integer = SampleRate / 10
+
+Type PlayPayload
+    frequency As Single
+End Type
+
+Function PlayCallback (InputBuff as Const Any Ptr, OutputBuff as Any Ptr, _
+                    FrameCount as CUlong, TimeInfo as Const PaStreamCallbackTimeInfo Ptr, _
+                    StatusFlags as PaStreamCallbackFlags, UserData as Any ptr) as Long
+
+    Dim as Long Ptr outBuff = OutputBuff
+    Dim as PlayPayload Ptr payload = UserData
+    Dim as Single frequency = payload->frequency
+    For i as Integer = 0 To FrameCount - 1
+        outBuff[i] = CLng(&h7FFFFFFF * 0.5 * Sin(2 * Pi * frequency * i / SampleRate))
+    Next
+    Function = paContinue
+End Function
+
+Sub Play(Frequency as Single, Duration as Single)
+    Dim as PaError pe
+    Dim as PaStream Ptr stream
+
+    Dim as PlayPayload payload
+    payload.frequency = Frequency
+
+    pe = Pa_OpenDefaultStream(@stream, 0, 1, paInt32, SampleRate, SineSamples, @PlayCallback, @payload)
+    If pe <> paNoError Then
+        Print "Error: "; Pa_GetErrorText(pe)
```

Comment:
`Pa_GetErrorText(pe)` here and a few other locations below are missing the dereference `*` to print the pointed to string.  i.e. `*Pa_GetErrorText(pe)`